app/views/clients/index.html.slim

```diff
@@ -2,19 +2,17 @@ h1= t('clients')
 nav.navbar.section-navigation
   hr
   ul.list-inline
-    li= button_link t_title(:new), new_client_path, 'success', dimension: 'sm'
-    - if policy(Client).destroy?
-      li= button_link t('need_accompanying'), need_accompanying_clients_url, dimension: 'sm'
+    li= button_link 'Klient erfassen', new_client_path, dimension: 'sm'
```

Comment:
Use translation for `'Klient erfassen'`?

Reply:
It was a team decision to drop translating the application from now on, as it is an unneccesary overhead. (the application is way to specific to AOZ, so it is quite unlikeley that ever anyone would use this in english, not doing a complete refactor/self customization of the application)

Reply:
Ok noted!